test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -40,6 +40,31 @@ namespace Npgsql.Tests
 {
     public class ConnectionTests : TestBase
     {
+
+        [OneTimeSetUp]
+        public void CreateTestFile()
+        {
+            // set up pgpass file with connection credentials
+            const string content = "*:*:*:npgsql_tests:npgsql_tests";
+            var pgpassFile = PgPassFile.GetPgPassFilePath();
+            var directory = Path.GetDirectoryName(pgpassFile);
+            if (directory != null)
+            {
+                Directory.CreateDirectory(directory);
+            }
+            File.WriteAllText(pgpassFile, content);
+        }
+
+        [OneTimeTearDown]
+        public void DeleteTestFile()
+        {
+            var pgpassFile = PgPassFile.GetPgPassFilePath();
+            if (File.Exists(pgpassFile))
+            {
+                File.Delete(pgpassFile);
```

Comment:
Also please move this setup code to the bottom of ConnectionTests.cs and wrap it in a Setup/Teardown region.